src/components/holofuel/layout/PrimaryLayout.js

```diff
@@ -0,0 +1,27 @@
+import React, { useContext } from 'react'
```

Comment:
can you put this in the pod pattern of `layout/PrimaryLayout/PrimaryLayout.js`, and then export it from `layout/PrimaryLayout/index.js`, so we can keep tests and css files etc tidy